backend/routes/follows.js

```diff
@@ -62,7 +62,7 @@ router.post("/follow", async (req, res) => {
 
         const follow = await Follow.findOne({ where: followData });
 
-        if (follow != null) {
+        if (follow !== null) {
```

Comment:
Curious, is there a reason you've changed it this way? It this never undefined?

Reply:
Should I put it to include undefined as well? The findOne returns an empty array if nothing is found.